src/main/kotlin/study/racingcar/entity/Car.kt

```diff
@@ -1,8 +1,10 @@
 package study.racingcar.entity
 
-class Car(private var moveCount: Int = 0) {
+class Car(private val carName: String, private var moveCount: Int = 0) {
     val currentPosition: Int
         get() = moveCount
+    val name: String
+        get() = carName
```

Comment:
음 이럴 경우 변수 선언이 중복되는 느낌이 있어요. private val로 선언할 게 아니라 매개변수로 선언해주면 이런 문제도 없을 것 같은데 어떨까요? 

```suggestion
class Car(carName: String, moveCount: Int = 0) {
    val currentPosition: Int = moveCount
    val name: String = carName
```